lib/http/HttpRequestEncoder.cpp

```diff
@@ -45,6 +45,7 @@ namespace MAT_NS_BEGIN {
         ctx->httpRequest->SetUrl(m_config.GetCollectorUrl());
 
         ctx->httpRequest->GetHeaders().set("Expect", "100-continue");
+        ctx->httpRequest->GetHeaders().set("CV", ctx->cvStr);
```

Comment:
This looks to be non-standard header content, not part of contract with Collector.

Reply:
we will remove this MIP Http delegate before sending to collector

Reply:
And how will this affect the non-windows clients ?

Reply:
Currently this will not be checked in for non-mip clients. It will be used for our specific scenarios.

Meanwhile Lalit it will be great if 1DS can expose LogManager for future perspective in dll

Reply:
Made required changes